code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -185,7 +185,6 @@
 /obj/item/storage/firstaid/tactical/populate_contents()
 	new /obj/item/reagent_containers/hypospray/combat(src)
 	new /obj/item/reagent_containers/applicator/dual/syndi(src) // Because you ain't got no time to look at what damage dey taking yo
-	new /obj/item/defibrillator/compact/combat/loaded(src)
```

Comment:
```
/obj/item/defibrillator/compact/loaded(src)
```
I'd say you should put a regular compact in this, if a nukie is going for a "medical" build, they should at least be given a defib of some sort. But that's just my balance opinion.

Reply:
I'd say maybe a handheld actually, something that can fix cardiac arrest in the event you somehow recover an ally who's in crit but hasn't exploded

Reply:
pretty sure syndicate nanites fix that. maybe we could add an injector of those?

Reply:
Added a compact handheld defib to the medkit.